### PR TITLE
Language fixes & bug fixes

### PR DESCRIPTION
**Pull request by ekaterina-git**

### CODE REVIEW

File: src/messages/ru/podium/view.php [hunk 1]
```diff
@@ -341,6 +341,7 @@
     'Sign in' => 'Войти',
     'Sign in to reply' => 'Залогиньтесь, чтобы ответить',
     'Sign out' => 'Выйти',
+    'Signature' => 'Подпись',
```
Comment (bizley) [inline src/messages/ru/podium/view.php]: This text has been removed in previous commit. There is 'IP' there instead (admin > logs). 
Please remove this line.

**Comment by bizley** [comment]
Wow, thank you. Some old code slipped through in last commit ;)
Could you remove this one line though?

**Comment by bizley** [comment]
BTW have you seen this already? ;) https://hacktoberfest.digitalocean.com/

File: src/messages/ru/podium/view.php (hunk 1 above)
Comment (ekaterina-git) [reply src/messages/ru/podium/view.php]: Thanks. I removed this line!

**Comment by ekaterina-git** [comment]
You're welcome.
Well, as a matter of fact I haven't, thanks for the info 😁

**Comment by bizley** [comment]
Cheers

**Pull request merged**